YMChat/index.html

```diff
@@ -9,143 +9,134 @@
     <meta name="mobile-web-app-capable" content="yes">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
     <style>
-         .no-js #loader { display: none;  }
-         .js #loader { display: block; position: absolute; left: 100px; top: 0; }
-         .se-pre-con {
-         position: fixed;
-         left: 0px;
-         top: 0px;
-         width: 100%;
-         height: 100%;
-         z-index: 9999;
-         background: url(https://thumbs.gfycat.com/ImpoliteLivelyGenet-max-1mb.gif) center no-repeat #fff;
-         }
-      </style>
+        .no-js #loader { display: none;  }
+        .js #loader { display: block; position: absolute; left: 100px; top: 0; }
+        .se-pre-con {
+            position: fixed;
+            left: 0px;
+            top: 0px;
+            width: 100%;
+            height: 100%;
+            z-index: 9999;
+            background-repeat: no-repeat;
+            background-position: center;
+        }
+
```

Comment:
please take updated index.html from sukrit.
Have asked him to make some changes in formatting